### PR TITLE
vendor: update docker/cli, docker/docker, containerd

### PR DESCRIPTION
**Pull request by thaJeztah**
### vendor: github.com/containerd/containerd v1.6.8

Relevant changes:

- Fix WWW-Authenticate parsing
    - fixes "Failed to parse Www-Authenticate if auth-param has empty value"

full diff: https://github.com/containerd/containerd/compare/v1.6.6...v1.6.8

### vendor: github.com/docker/go-units v0.5.0

full diff: https://github.com/docker/go-units/compare/v0.4.0...v0.5.0

### vendor: docker v20.10.3-0.20220831131523-b5a0d7a188ac (22.06 branch)

full diff: https://github.com/moby/moby/compare/a60b458179aa...b5a0d7a188aced307cf3902dfd765b9a67c4d9dd

### vendor: docker/cli v20.10.3-0.20220831164732-67cc8b1fd88a (master / 22.06-dev)

full diff: https://github.com/docker/cli/compare/418ca3b4d46f...67cc8b1fd88aea06690eaf3e5d56acd68a0178d2

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -104,6 +104,7 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0 // indirect
 	github.com/moby/locker v1.0.1 // indirect
 	github.com/moby/spdystream v0.2.0 // indirect
+	github.com/moby/sys/sequential v0.0.0-20220829095930-b22ba8a69b30 // indirect
```
Comment (thaJeztah) [inline go.mod:107]: I tagged a v0.5.0 release for this; let me update

Comment (thaJeztah) [reply go.mod]: done 👍

**Review by tonistiigi — COMMENTED** [review]
We usually update these dependencies through buildkit(especially contained that doesn't have almost any dependency for buildx) so they versions are not different for same libraries.

**Comment by thaJeztah** [comment]
Yes, was working on a branch for BuildKit as well; just opened a PR https://github.com/moby/buildkit/pull/3075

Overall, it may become a bit more complicated, as;

- buildx depends on docker/cli and buildkit
- buildkit depends on moby (22.06 branch)
- moby 22.06 branch depends on buildkit (v0.10)
- docker/cli depends on moby (22.06 branch)

Which means there's a dependency on both BuildKit master and BuildKit v0.10, so we must be sure that BuildKit master is backward compatible with BuildKit v0.10. Currently, moby and docker/cli don't have a `go.mod`, so we can still be somewhat flexible, but once they do, go modules will enforce these versions (or overrides are needed).

File: go.mod [hunk 2]
```diff
@@ -6,12 +6,12 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.15.5
 	github.com/compose-spec/compose-go v1.5.0
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd v1.6.6
+	github.com/containerd/containerd v1.6.8
```
Comment (AkihiroSuda) [inline go.mod:9]: Now v1.6.10

**Comment by crazy-max** [comment]
Done in https://github.com/docker/buildx/pull/1420 (https://github.com/docker/buildx/pull/1420/commits/36e663edda306ee79a4b6b89d8a2ba2968cdc385)

**Pull request closed without merging**